js/foundation.offcanvas.js

```diff
@@ -364,9 +364,9 @@ class OffCanvas extends Plugin {
   _stopScrollPropagation(event) {
     let elem = this; // called from event handler context with this as elem
     let parent; // off-canvas elem if called from inner scrollbox
-    let up = event.pageY < elem.lastY;
+    let up = event.touches[0].pageY > elem.lastY;
     let down = !up;
-    elem.lastY = event.pageY;
```

Comment:
`event.pageY` is a valid property of jQuery events. See https://api.jquery.com/event.pagey/